Assets/Scripts/Game/WeaponManager.cs

```diff
@@ -481,7 +481,22 @@ private void MeleeDamage(FPSWeapon weapon)
 
                 // TODO: Use correct damage based on weapon and swing type
                 // Just using fudge values during development
```

Comment:
Also, since the manual refers to these bonuses as being applied to damage "potential", does that suggest that they are only applied to the upper bound of the damage range?